File: app/src/main/java/org/mifos/selfserviceapp/models/beneficary/BeneficiaryUpdatePayload.java
package org.mifos.selfserviceapp.models.beneficary;

import com.google.gson.annotations.SerializedName;

/**
 * Created by dilpreet on 16/6/17.
 */

public class BeneficiaryUpdatePayload {

    @SerializedName("name")
    String name;

    @SerializedName("transferLimit")
    int transferLimit;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getTransferLimit() {
        return transferLimit;
    }

    public void setTransferLimit(int transferLimit) {
        this.transferLimit = transferLimit;
    }
}

File: app/src/main/java/org/mifos/mobilebanking/models/beneficary/BeneficiaryUpdatePayload.kt
package org.mifos.mobilebanking.models.beneficary

import com.google.gson.annotations.SerializedName

/**
 * Created by dilpreet on 16/6/17.
 */

data class BeneficiaryUpdatePayload @JvmOverloads constructor(

    @SerializedName("name") var name: String? = null,

    @SerializedName("transferLimit") var transferLimit: Int = 0)
